sw/device/silicon_creator/lib/manifest.h

```diff
@@ -261,6 +300,67 @@ typedef struct manifest_digest_region {
   size_t length;
 } manifest_digest_region_t;
 
+/**
+ * Required manifest extension header.
+ */
+typedef struct manifest_ext_header {
+  uint32_t identifier;
+} manifest_ext_header_t;
+
+/**
+ * Extension identifiers.
+ */
+enum {
+  /**
+   * ASCII "EXT0.
+   */
+  kManifestExtIdSpxKey = 0x30545845,
+  /**
+   * ASCII "EXT1.
+   */
+  kManifestExtIdSpxSignature = 0x31545845,
```

Comment:
Is it OK that the Hamming distance between these is only 1 bit?

Reply:
Good catch!
I'm also on the fence about this. During our last meeting, @moidx recommended having identifiers that are easy to check in a hex dump for ease of debugging. We have a meeting today and I was planning to bring this up.

Reply:
The marker doesn't have to be the manifest ID if there is a way for the attacker to leverage the hamming distance here. 

It is ok to have these as two separate fields (`id`, `name`). I still like to have the ASCII identifier to quickly look for things in hex dumps.

Reply:
Thanks for chiming in @moidx, do you think if it would be enough to split 32-bit as: 16-bit high HW, 16-bit ASCII, or 8-bit high HW, 24-bit ASCII?

(asking because we have N entries)

Reply:
We talked about this offline and we will use identifiers with high HD here, the header struct will have another field for the ascii name.
@moidx PLMK if you disagree.
Thanks!

Reply:
this SGTM. Thanks @alphan!